tests/labhub_test.py

```diff
@@ -180,6 +180,27 @@ def test_assign_cmd(self):
 
         testbot.assertCommand(cmd.format('coala', 'a', '23'),
                               'You\'ve been assigned to the issue')
+        
+        # no assignee, newcomer, initiatives/gci
+        mock_maint_team.is_member.return_value = False
+        mock_dev_team.is_member.return_value = False
+        mock_issue.labels = 'initiatives/gci',
+        mock_issue.assignees = tuple()
+        testbot.assertCommand(cmd.format('coala', 'a', '23'),
+                              'You are not eligible to be assigned ' \
```

Comment:
unnecessary `\`
python has implicit string concatenation

Reply:
Thanks, I'm don't know how you know things like this. I get to learn so much from you guys. This is the best community out there. You guys are the best. @jayvdb @Makman2 @nvzard